### PR TITLE
Veda protocol addition

### PR DESCRIPTION
**Pull request by philipjames44**
##### Name (to be shown on DefiLlama): 
Veda

##### Twitter Link:
https://x.com/veda_labs

##### List of audit links if any:
https://github.com/Se7en-Seas/boring-vault/tree/main/audit
https://0xmacro.com/library/audits/sevenSeas-1.html
https://0xmacro.com/library/audits/sevenSeas-2.html
https://0xmacro.com/library/audits/sevenSeas-3.html

##### Website Link:
https://veda.tech/

##### Logo (High resolution, will be shown with rounded borders):
veda_logo

##### Current TVL:
~$721M

##### Chain:
Ethereum

##### Short Description (to be shown on DefiLlama):
Veda is the first native yield layer. Veda tokenizes the most complex DeFi interactions, granting protocols the power to embed yield for a superior UX.

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Yield

##### forkedFrom (Does your project originate from another project):
No

##### methodology (what is being counted as tvl, how is tvl being calculated):
TVL is calculated as the sum of deposits invested into the strategy, deposits waiting to be invested, and yield waiting to be reinvested or redistributed across all vaults.

##### Github org/user (Optional, if your code is open source, we can track activity):
https://github.com/Veda-Labs

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/veda exports TVL: 
        
 
 
    ethereum                  719.27 M
    
    total                    719.27 M

File: projects/veda/constants.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const ADDRESSES = require("../helper/coreAssets.json");
+
+const legacyVaults = [
+  {
+    id: "0xeA1A6307D9b18F8d1cbf1c3Dd6aad8416C06a221",
```
Comment (g1nt0ki) [inline projects/veda/constants.js:5]: this is ether.fi liquid vault right? why are you counting it as your tvl?

Comment (philipjames44) [reply projects/veda/constants.js:5]: The Veda protocol is the infrastructure that is powering EtherFi Liquid

Comment (g1nt0ki) [reply projects/veda/constants.js:5]: can you explain bit more, how it works? who deployed/controls the contracts? we can count the tvl as part of only one dapp, not both

Comment (philipjames44) [reply projects/veda/constants.js:5]: veda isn't a dapp, it's the smart contract infrastructure that the etherfi liquid product is built on. The general idea of veda is a protocol that enables the creation of tokenized yield products. Liquid is the first instance of a tokenized yield product built on veda but there are more to come

Comment (g1nt0ki) [reply projects/veda/constants.js:5]: yes, but what does smart contract infrastructure mean? 

Is any of the vaults in this PR deployed/controlled by your team?

Comment (philipjames44) [reply projects/veda/constants.js:5]: The smart contracts were deployed by our team however the question of “control” is a more involved one.   High level Veda is a system that allows protocols to offer yield products which are optimized by off-chain strategists. The Veda smart contracts constraint the role of the strategist along with potential other service providers through a set of on-chain and off-chain checks and balances. The smart contracts have various roles that the protocol integrator (EtherFi in this case) can use to add security or custom controls, for example changing out the strategist.

In the case of EtherFi Liquid, the strategist is Seven Seas and EtherFi itself has an administration role in the contracts. The security model, roles, strategist rebalancing process, etc. are part of the Veda protocol.

An existing analogy for something that’s already integrated in DeFiLlama is the [Swell Vault](https://defillama.com/protocol/swell-vault#information) and [DIVA Vault](https://defillama.com/protocol/diva#information) both built by Enzyme. Both Swell and DIVA are independently reporting these vaults in their TVL, and the TVL from both vaults are also included in [Enzyme’s total TVL](https://defillama.com/protocol/enzyme-finance#information).

Comment (philipjames44) [reply projects/veda/constants.js:5]: Hi @g1nt0ki just following up. Let me know if this makes sense

Comment (philipjames44) [reply projects/veda/constants.js:5]: Hey @g1nt0ki just wanted to check in here again

Comment (g1nt0ki) [reply projects/veda/constants.js:5]: sorry, for the delay, we are still discussing how to handle this

Comment (g1nt0ki) [reply projects/veda/constants.js:5]: the issue remains that we cant count the tvl in same contract as part of two different projects, can you check with etherfi? we can maybe delist them and list this

**Comment by g1nt0ki** [comment]
closing this for now, till this issue is resolved: https://github.com/DefiLlama/DefiLlama-Adapters/issues/10566

**Pull request closed without merging**